### DISCUSSION TITLE
如何分流代理？

### DISCUSSION THREAD

**Discussion by MMMMMoris** [vernesong/OpenClash#2700] [Q&A]
我有两个无线信号，一个是2.4G的OpenWRT-2.4G，一个是5G的openWRT-5G，我在想如何在openWRT-5G上代理，但是openWRT-2.4G不用代理，直连国内的，请问如何设置能实现这种功能？

**Comment by theVincentWei** [comment]
是不是可以通过网卡来分流，2.4G和5G是一个网卡吗？

**Reply by MMMMMoris** [reply]
是两张不同的网卡

**Reply by theVincentWei** [reply]
我看clash的配置是不支持按网卡分流，但是openwrt的防火墙可以设置成只把来自5G的流量转发到openclash的代理端口，2.4G的流量直接到WAN口，但怎么配置我不太清楚😂，具体怎么转发的这块我也还在研究，还在等待别人的解答

**Reply by kouzili77** [reply]
有方法了吗？

**Reply by theVincentWei** [reply]
添加route路由规则就行
